x/gamm/keeper/multihop_test.go

```diff
@@ -176,7 +176,7 @@ func (suite *KeeperTestSuite) TestSimpleMultihopSwapExactAmountOut() {
 
 			// Ratio of the oken out should be between the before spot price and after spot price.
 			sp := tokenInAmount.ToDec().Quo(test.param.tokenOut.Amount.ToDec())
-			fmt.Println(spotPriceBefore.String(), spotPriceAfter.String(), sp.String())
+			fmt.Println(spotPriceBefore.String(), spotPriceAfter.String(), sp.String()) // TODO: Remove?
```

Comment:
Yeah, woops. I think I added this for some debugging at some point

Reply:
```suggestion
```